x-pack/plugins/observability/server/types/rest_specs/slo.ts

```diff
@@ -81,11 +88,18 @@ const updateSLOResponseSchema = t.type({
 
 type CreateSLOParams = t.TypeOf<typeof createSLOParamsSchema.props.body>;
 type CreateSLOResponse = t.TypeOf<typeof createSLOResponseSchema>;
-type GetSLOResponse = t.TypeOf<typeof getSLOResponseSchema>;
+type GetSLOResponse = t.OutputOf<typeof getSLOResponseSchema>;
```

Comment:
What does this change mean? Why we don't need something similar for `CreateSLOResponse`?

Reply:
when the schema is made of custom type, we want to return the representation of such type. e.g. the durationType is a class that when serialized becomes `{ unit: "w", value: 1 }` when we want to keep the string representation `"1w"` as provided in input.
Therefore using OutputOf when we have custom type allows to have a type with the correct representation.

For CreateSLOResponse I haven't done it since its type is just a `{ id: t.string }` which has the same representation, e.g. a string is a string. 
I definitely could change it and nothing will break since `TypeOf` and `OutputOf` a t.string is the same, e.g. a string